warehouse/ingest-configuration/src/main/resources/config/edge-definitions.xml

```diff
@@ -1220,7 +1220,7 @@ http://www.springframework.org/schema/util/spring-util-4.0.xsd">
 
     </bean>
 
-    <bean id="default.protobufedge.table.blacklist.values" class="org.springframework.beans.factory.config.SetFactoryBean">
+    <bean id="default.protobufedge.table.disallowlist.values" class="org.springframework.beans.factory.config.SetFactoryBean">
```

Comment:
I thought we were saving these changes for your other PR?  This was good to go, but this commit dismissed all of the approvals...  Also, isn't this change meaningless without also changing EDGE_TABLE_BLACKLIST_VALUES in ProtobufEdgeDataTypeHandler?

Reply:
This is me confusing two different PRs and now needing to revert the changes made - totally unintentional.